.github/workflows/cspell5-update-dependencies.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           private_key: ${{ secrets.AUTOMATION_PRIVATE_KEY }}
       - name: Create Pull Request
         if: env.git_status == 'dirty'
-        uses: peter-evans/create-pull-request@b4d51739f96fca8047ad065eccef63442d8e99f7 # v4
+        uses: peter-evans/create-pull-request@d7db273d6c7206ba99224e659c982ae34a1025e3 # v4
```

Comment:
```suggestion
        uses: peter-evans/create-pull-request@v4
```

Reply:
@nschonni,

Please see [GitHub Security: Using third-party actions](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions).